ui/widgets/nuxeo-date-picker.js

```diff
@@ -203,7 +203,8 @@ import { I18nBehavior } from '../nuxeo-i18n-behavior.js';
     }
 
     _getValidity() {
-      return this.$.date.validate() && (this.required ? !!this.value : true);
+      const formattedDate = this._moment(this.value).format(moment.localeData().longDateFormat('L'));
```

Comment:
nit: I think we could reduce some code duplication using something like:
```suggestion
      const formattedDate = this.$.date.i18n.formatDate(this.value);
```
We already set the `i18n.formatDate` method with the very some logic in `ready`.